Fixing links to download app signing scripts (#755)
mattermost/mattermost-developer-documentation: site/content/contribute/mobile/unsigned/android.md

> 3. [JQ](https://stedolan.github.io/jq/) is like sed for JSON data - you can use it to slice, filter, map, and transform structured data with the same ease that `sed`, `awk`, and `grep` let you work with text.

Reviewer: ```suggestion
3. [JQ](https://stedolan.github.io/jq/) is like `sed` for JSON data - you can use it to slice, filter, map, and transform structured data with the same ease that `sed`, `awk`, and `grep` let you work with text.
```

4. Android SDK as described in the [Developer Setup](/contribute/mobile/developer-setup/#additional-setup-for-android).

> 5. Setup keys and Google Services as described in steps 2, 3, 4 & 6 of the [Build your own App guide](/contribute/mobile/build-your-own/android/#build-preparations).

Reviewer 2: ```suggestion
5. Set up keys and Google Services as described in steps 2, 3, 4, and 6 of the [Build your own App guide](/contribute/mobile/build-your-own/android/#build-preparations).
```

> 6. [sign-android](/scripts/sign-android) script to sign the Android app.

Reviewer 2: Is this a link to a file or a section in the docs?

#### Sign the Mattermost Android app

> Now that all requirements are met, it's time to sign the Mattermost app for Android. Most of the options of the signing tool are optional but you should use your own `package identifier`, `google services settings` and change the `display name`.

Reviewer 2: ```suggestion
Now that all requirements are met, it's time to sign the Mattermost app for Android. Most of the options of the signing tool are optional but you should use your own `package identifier`, `google services settings`, and change the `display name`.
```

> Now that all requirements are met, it's time to sign the Mattermost app for Android. Most of the options of the signing tool are optional but you should really be using your own `package identifier`, `google services settings` and maybe change the `display name`.

Reviewer: ```suggestion
Now that all requirements are met, it's time to sign the Mattermost app for Android. Most of the options of the signing tool are optional but you should use your own `package identifier`, `google services settings` and change the `display name`.
```

* Create a folder that will serve as your working directory to store all the needed files.
* Download the [sign-android](/scripts/sign-android) script and save it in your working directory.
* Download the [Android unsigned build](https://github.com/mattermost/mattermost-mobile/releases) and save it in your working directory.

> * Open a terminal to your working directory and make sure the `sign-android` script it is executable.

Reviewer 2: ```suggestion
* Open a terminal to your working directory and make sure the `sign-android` script is executable.
```
